### PR TITLE
Makes mindslaves appear on the round end antagonist listing

### LINKED ISSUE #14156: Mindslaved people are not shown in the end of round summary
**Issue Description**:
Mindslaved players were not shown at the end of the round summary

**What did you expect to happen**: 
Them to be shown as antags

**What happened instead**:
None were there.

**Why is this bad/What are the consequences**:
It indicates that people who were mindslaved were self antagging. Which can lead to false positive ahelps and what not.

**Steps to reproduce the problem**:
Mindslave somebody. 
Have the round end

**When did the problem start happening**:
Unsure. First day of it me seeing it was the 27-8-2020

**Extra information**:
The round type was vamp traitor.

### PR DESCRIPTION
**Pull request by SteelSlayer**
## What Does This PR Do
Makes mindslaves appear on the round end antagonist listing. Also, slightly tweaks the look of the end of round traitor name listing. It now includes their starting job as well as colored text for the "survived" or "died" bit. 

## Why It's Good For The Game
Other game modes with potential thralls/mindslaves also list them out at the end of the round, such as vampire thralls.

Closes #14156. 

## Images of changes
dreamseeker_ghYqjUN0A8

Edit: Updated PR to make the "The mindslaves were" text the same size as the traitor text, as I realized it was too big. I'm not going to take another picture though because I'm lazy.

## Changelog
:cl:
tweak: Slightly tweaked the look of the end of round traitor name listing. It now includes their starting job.
tweak: Syndicate mindslaves will now show up in the end of round antagonist listing.
/:cl:

### CODE REVIEW

**Comment by AffectedArc07** [comment]
Maybe add some text of who they were slaved by?

**Comment by SteelSlayer** [comment]
Added the slaved by text. Updated picture in the description.

File: code/game/gamemodes/traitor/traitor.dm [hunk 1]
```diff
@@ -82,22 +82,10 @@
...
-					text += "died"
-				else
-					text += "survived"
-				if(traitor.current.real_name != traitor.name)
-					text += " as [traitor.current.real_name]"
-			else
-				text += "body destroyed"
-			text += ")"
-
+			text += printplayer(traitor)
```
Comment (Kyep) [inline code/game/gamemodes/traitor/traitor.dm:88]: How does this change what info is printed for traitors?

File: code/game/gamemodes/traitor/traitor.dm [hunk 2]
```diff
@@ -137,6 +125,14 @@
 				text += "<br><font color='red'><B>The [special_role_text] has failed!</B></font>"
 				feedback_add_details("traitor_success","FAIL")
 
+		if(length(SSticker.mode.implanted))
+			text += "<br><br><FONT size = 2><B>The mindslaves were:</B></FONT><br>"
```
Comment (Kyep) [inline code/game/gamemodes/traitor/traitor.dm:129]: Don't they already show as traitors with no objective but 'protect'?

Comment (SteelSlayer) [reply code/game/gamemodes/traitor/traitor.dm:129]: They do not. Previously there was nothing shown.

File: code/game/gamemodes/traitor/traitor.dm (hunk 1 above)
Comment (SteelSlayer) [reply code/game/gamemodes/traitor/traitor.dm:88]: The picture in the description is the complete set of information that is shown at the end of a traitor round.

**Review by Dumbdumn5 — APPROVED** [review]
Decent QoL change if things with Travis go well!

**Pull request merged**